devtools-ci-index.yaml

```diff
@@ -556,6 +556,10 @@
                 - openshift-connector
     triggers:
         - github
+        - github-pull-request:
```

Comment:
If we would keep just the github trigger, only way to test job would be pushing to Che repository, what is IMHO the bad idea - to test that job is working, push some commit, if something has to be fixed in job, do another push to the repo. We would pollute repo with nonsense commits. If that is acceptable for redhat-developer/rh-che repo, I can change it back just to the github trigger.

Another thing, why I think it is non-sense to trigger a verification job on github trigger is that triggering a verification job on something, what is already in master does not make sense - at first, code should pass unit tests, then some sort of integration tests and after that it should be merged to master/other branch we are using. I know, we are going to have it for PRs, but I have quite different idea about promoting builds - how it should be done, when it should happen, what are the processes.